cpp/src/parquet/encoding_benchmark.cc

```diff
@@ -199,6 +199,107 @@ static void BM_PlainDecodingFloat(benchmark::State& state) {
 
 BENCHMARK(BM_PlainDecodingFloat)->Range(MIN_RANGE, MAX_RANGE);
 
+static void BM_PlainEncodingSpacedBoolean(benchmark::State& state) {
```

Comment:
That's not how booleans are encoded in Parquet (i.e. 1-bit data), is it?

Reply:
Indeed it's the one bit encoding. The put space include two stage, first is to filter the value as the valid bit map, later is call the the boolean encode routine on the valid data which remove spaced values already.

Reply:
You mean boolean encoding converts a bitmap to an array of `bool`?